deepgram/transcription.py

```diff
@@ -239,7 +240,8 @@ async def _receiver(self) -> None:
             try:
                 body = await self._socket.recv()
                 self._queue.put_nowait((True, body))
-            except Exception as exc:
```

Comment:
Should this still catch Exception after the new exception catch just in case some other error happens?

Reply:
I personally think we should not catch all exceptions. My guess is that the original code caught all exceptions specifically so that the `ConnectionClosedOk` exception was handled correctly. Below is more info on my opinion:

>This PR also makes a small change to the exception handling. Rather than catching all websocket closures (and other any exceptions!) by using except Exception as exc, I replaced the error handling with a more explicit catch: except websockets.exceptions.ConnectionClosedOK:. This change will result in errors that are not ConnectionClosedOK bubbling up to the user. I think this is appropriate because any other websocket closures are errors that the user may want to handle or be aware of. I was unable to produce any situations where a different exception was thrown, but we may get messages from customers who find edge cases.

Reply:
@DamienDeepgram what makes you want to catch and log these specific errors rather than throwing them?

The new try/except clauses catches the 1011 "error" and closes the websocket cleaning when that response code is received. Deepgram returns the 1011 "error" as a notification to the user that Deepgram closed the socket due to inactivity, which the previous version of the Python SDK considered a normal piece of the workflow (i.e. not an exceptional case).

The new code handles the 1011 "error" in the same way, but raises other errors to the user. I think that makes sense because those other errors are real errors (whereas the 1011 error is a simply closure due to inactivity). But I'm curious to hear what you're thinking.